lib/src/main/java/growthbook/sdk/java/GBFeaturesRepository.java

```diff
@@ -246,7 +245,15 @@ private void onSuccess(Response response) throws FeatureFetchException {
         }
     }
 
+    private void refreshSuccessfullyCallbacks(String featuresJson) {
+        this.refreshCallbacks
+                .forEach(featureRefreshCallback -> featureRefreshCallback.onRefresh(featuresJson));
```

Comment:
Style: Can you use the existing style (add braces).

Reply:
corrected)